src/momento/aio/simple_cache_client.py

```diff
@@ -1,9 +1,17 @@
 from types import TracebackType
 from typing import Optional, Union, Type
 
-from ._scs_control_client import _ScsControlClient
-from ._scs_data_client import _ScsDataClient
-from .._utilities._data_validation import _validate_request_timeout
+try:
+    from ._scs_control_client import _ScsControlClient
+    from ._scs_data_client import _ScsDataClient
+    from .._utilities._data_validation import _validate_request_timeout
+except ImportError as e:
+    if e.name == 'cygrpc':
+        import sys
+        print('There is an issue on M1 macs between GRPC native packaging and Python wheel tags. See https://github.com/grpc/grpc/issues/28387', file=sys.stderr)
+        print('TO WORK AROUND, please install Rosetta 2 and re-run with:', file=sys.stderr)
+        print('arch -x86_64 {} {}'.format(sys.executable, *sys.argv), file=sys.stderr)
```

Comment:
Hmmm I'm not entirely sure this is what we want, although this may also be something to do with how I tried this - tried doing:

```
$ /opt/homebrew/opt/python@3.9/bin/python3.9 example.py
There is an issue on M1 macs between GRPC native packaging and Python wheel tags. See https://github.com/grpc/grpc/issues/28387
TO WORK AROUND, please install Rosetta 2 and re-run with:
arch -x86_64 /opt/homebrew/opt/python@3.9/bin/python3.9 example.py
Traceback (most recent call last):
```

but that resulted in
```
$ arch -x86_64 /opt/homebrew/opt/python@3.9/bin/python3.9 example.py
arch: posix_spawnp: /opt/homebrew/opt/python@3.9/bin/python3.9: Bad CPU type in executable
```

But I guess if you were using just `python3` or something it might be fine?

Separately, it might be nice to make this error a little more obvious, like indent the command, maybe something else?

Reply:
I've addressed this. Added indents & formatting.

I cannot tell which python the user should execute - so I've enhanced the message.